src/i18n.js

```diff
@@ -71,6 +71,10 @@ export function setupI18n() {
     document.documentElement.lang = lang;
     localize('[data-i18n]');
   });
+  const templates = SUPPORTED_LANGUAGES.map(lang => html`
+    <div class="item lang-${lang}" @click="${() => i18next.changeLanguage(lang)}">${lang.toUpperCase()}</div>
+  `);
+  render(templates, document.getElementById('langs'));
```

Comment:
This should not be done here. See the other comment.

Reply:
addressed it in https://github.com/swissgeol/ngm/pull/667/commits/6f51e81fa90278d807f3623d6175286e9729f4ac